testplan/web_ui/testing/src/Nav/TreeView.js

```diff
@@ -4,15 +4,37 @@ import TreeView from '@material-ui/lab/TreeView';
 import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
 import ChevronRightIcon from '@material-ui/icons/ChevronRight';
 import TreeItem from '@material-ui/lab/TreeItem';
+import { css, StyleSheet } from 'aphrodite';
 import Column from './Column';
 import NavEntry from './NavEntry';
 import { applyAllFilters } from './navUtils';
-import { STATUS } from "../Common/defaults";
+import { STATUS, MEDIUM_GREY } from "../Common/defaults";
 import { CATEGORIES } from "../Common/defaults";
 import { generatePath } from 'react-router';
 import { NavLink } from 'react-router-dom';
 import TagList from './TagList';
+import { makeStyles } from '@material-ui/core/styles';
 
+const createTreeViewStyles = makeStyles({
+  treeItemLabel: {
+    padding: '5px 0px',
+    '&:hover': {
+      backgroundColor: MEDIUM_GREY
+    },
+    // WORKAROUND:
+    // if display: flex (default) then entry icons get cut off in case a long test case name
+    // if display: block then expand / collapse icons get rendered in a seperate row
+    width: '90%',
```

Comment:
you just need to set "overflow: hidden;" in .MuiTreeItem-label

Reply:
You are a genius! 😀